### PR TITLE
Add some base CODEOWNERS

### PR DESCRIPTION
**Pull request by lleyton**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Package maintainers
+/anda/lib/libbismuth @lainsce
```
Comment (windowsboy111) [inline CODEOWNERS:2]: add lainsce to terrapkg members

**Review by windowsboy111 — CHANGES_REQUESTED** [review]
I think we should add the pipeline files
and some other files

(?)

**Comment by lleyton** [comment]
Yeah, feel free to push to this branch

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Package maintainers
```
Comment (windowsboy111) [inline CODEOWNERS:1]:
```suggestion
# Repository
/.github @lleyton 
/.devcontainer @lleyton 
/.gitignore @windowsboy111 
/CODE_OF_CONDUCT.md @lleyton 
/CONTRIBUTING.md @lleyton 
/LICENSE @lleyton 
/README.md @windowsboy111 
/SECURITY.md @lleyton 
/anda.hcl @lleyton 
/comps.xml @lleyton 
/requirements.txt @windowsboy111 
/update.sh @windowsboy111 

# Package maintainers
```

**Review by windowsboy111 — APPROVED** [review]

**Review by lainsce — APPROVED** [review]
lgtm

**Pull request merged**